libs/test-common/src/main/java/org/codice/ddf/test/common/ComponentTestRunner.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.test.common;
+
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.model.InitializationError;
+import org.ops4j.pax.exam.junit.PaxExam;
+
+/**
+ * JUnit test runner that dumps extra debug information to the console when a Pax Exam test
+ * container fails to start.
+ */
+public class ComponentTestRunner extends PaxExam {
```

Comment:
❓ Is this specific to component or could it be used for any PaxExam test cases ... itests or otherwise? If so then should we rename it and make it more generic? Also should we modify all our itests and tests that Tracy is doing to use that?

Reply:
Good points. I wanted to keep the runner specific to component tests for now so it can evolve differently than other runners we could have (e.g., integration test one), and also not expose the fact that it relies on Pax Exam. However, since the tests themselves have to know about Pax Exam anyways, I agree that abstracting that last part probably doesn't buy us anything. How about renaming the class `PaxExamComponentTestRunner`? Thoughts, suggestions?